examples/vae/ss_vae_M2.py

```diff
@@ -41,6 +44,7 @@ def __init__(self, output_size=10, input_size=784, z_dim=50, hidden_layers=(500,
         self.z_dim = z_dim
         self.hidden_layers = hidden_layers
         self.epsilon_scale = epsilon_scale
+        self.config_enum = config_enum == 'parallel'
```

Comment:
isn't this trail of args confusingly named? this becomes allow_broadcast?

Reply:
I agree this is confusing. Maybe instead set this to:
```
self.config_enum = config_enum if config_enum else None
```

Reply:
i renamed `self.config_enum` to `allow_broadcast` which is a flag based on if `enum` is parallel, which comes from the cla. `config_enum` is a string used in `config_enumerate`, and we only want to allow broadcasting for parallel enum, not sequential.  let me know if this is less confusing.